include/configparser.h

```diff
@@ -9,7 +9,7 @@
 namespace newsboat {
 
 enum class action_handler_status {
-	OK = 0,
+	VALID = 0,
```

Comment:
Just curious: how did you find the clash? Did it cause any errors?

Reply:
Yep, I think it was during adding ncurses.h to newsboat.cpp. I got a strange error and pinned it down to this include. Then I found out there's OK defined in ncurses.h (I think).